src/pages/StatisticsChart/activities/AddressBalanceRank.tsx

```diff
@@ -44,7 +45,8 @@ const useOption = () => {
       tooltip: !isThumbnail
         ? {
             trigger: 'axis',
-            formatter: (dataList: any) => {
+            formatter: dataList => {
+              assertIsArray(dataList)
```

Comment:
Is the exception thrown from `assertIsArray` handled by the library?

Reply:
they're not handled, the errors are expected not to throw

Reply:
I didn't get it.
If `dataList` is definitely an array, the assertion `assertIsArray` is useless, otherwise, an exception would be thrown by `assertIsArray` and is not handled anywhere.

Reply:
The signature of `formatter` is `(value: Format | Format[]) => string` 

In runtime, the `dataList` has type of `Format[]`, but type system thinks it could also be a `Format`

The assertion narrows down it's type from `Format | Format[]` to `Format[]`.

And if `assertIsArray` throws in runtime, the error is not handled anywhere